webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -47,9 +56,30 @@ export default class PostMessageView extends React.Component {
         return false;
     }
 
+    isPostEdited() {
+        return this.props.post.update_at && this.props.post.update_at > this.props.post.create_at;
+    }
+
     render() {
+        const {formatMessage} = this.props.intl;
+
         if (!this.props.enableFormatting) {
-            return <span>{this.props.message}</span>;
+            return (
+                <span>
+                    {this.props.post.message}
+                    {
+                        this.isPostEdited() ?
+                            <span className='edited'>
+                                &nbsp;
+                                <FormattedMessage
```

Comment:
@enahum, Or I can use something [html-to-react](https://www.npmjs.com/package/html-to-react) to remove use of `dangerouslySetInnerHTML` and then would also be able to remove use of `injectIntl`, `defineMessages` and ``formatMessage` and would be able to use `<FormattedMessage>`. Shall I do that?